src/main/java/com/google/sps/data/Route.java

```diff
@@ -51,10 +61,33 @@ public Route(long routeId, String routeName, boolean isPublic, long startHour, l
     this.startMinute = startMinute;
   }
 
+  public Route(
+      long routeId,
+      String routeName,
+      boolean isPublic,
+      boolean isCompleted,
+      long startHour,
+      long startMinute,
+      long numberOfRatings,
+      double sumOfRatings) {
+    this.routeId = routeId;
+    this.routeName = routeName;
+    this.isPublic = isPublic;
+    this.isCompleted = isCompleted;
+    this.startHour = startHour;
+    this.startMinute = startMinute;
+    this.numberOfRatings = numberOfRatings;
+    this.sumOfRatings = sumOfRatings;
+  }
+
   public void setRouteId(long routeId) {
     this.routeId = routeId;
   }
 
+  public void setUserAccess(int userAccess) {
```

Comment:
This should also take an enum type as an input parameter.

Reply:
Is it true that if I initialise enum in the class file then I'll be able to use that enum everywhere where I can use that class?

Reply:
It is probably best to create a new file for the enum under the data directory. You can then reference the enum from the other classes in the same way you would reference a class.